src/d3a/models/strategy/external_strategies/storage.py

```diff
@@ -80,6 +80,8 @@ def _delete_offer_impl(self, arguments, response_channel):
                 self.offers.remove_offer_from_cache_and_market(self.market, to_delete_offer_id)
             self.state.offered_sell_kWh[self.market.time_slot] -= \
```

Comment:
Are you sure that this is the correct formula ? I was under the impression that offered_sell_kWh was the total amount of energy of the posted offers, therefore the `-=` should be replaced with `=` here. Please correct me if I have not understood the storage state operation correctly.